pkg/ip/ipforward.go

```diff
@@ -18,10 +18,12 @@ import (
 	"io/ioutil"
 )
 
+// EnableIP4Forward enables IPv4 forwarding
```

Comment:
I don't follow. Besides ignoring the golint recommendation, are there any other options? Never heard about golin filtering before.